dist/s_docs

```diff
@@ -145,14 +145,13 @@ EOF
 		e=1
 	}
 
-	# Add optional extras
-	EXTRAS="../lang/java/src/com/wiredtiger/db"
+	EXTRAS=""
 	EXTRA_INPUT=""
 	for f in $EXTRAS ; do
 		[ -e "$f" ] && EXTRA_INPUT="$EXTRA_INPUT ../$f"
 	done
 
-	# Run again to generate the full doc set with Python and Java.
+	# Run again to generate the full doc set with Python.
 	[ "$additional_languages" -eq 1 ] && [ "x$EXTRA_INPUT" != "x" ] && (
 	cd ../src/docs &&
 		(eval cat Doxyfile $filter ; cat <<EOF
```

Comment:
Since EXTRAS is null, I think the second run of doxygen becomes a no-op - that is, same as the first run.  (Despite the comment, we don't generate any extra doc for python).  It's probable that from `EXTRAS=` to `doxygen -` can be deleted.  To verify that, it would be good compare the docs directory with that change vs docs directory generated with your current branch.

Reply:
I didn't find any difference, so removed the second execution.